From 7ab4483de673e313450286c8de21fb00ee3c3616 Mon Sep 17 00:00:00 2001
From: Jeff Layton <jlayton@redhat.com>
Date: Mon, 21 Apr 2014 12:17:55 -0400
Subject: [net] sunrpc: remove KERN_INFO from dprintk() call sites

Message-id: <1398082678-3620-2-git-send-email-jlayton@redhat.com>
Patchwork-id: 78973
O-Subject: [RHEL7.1 PATCH 1/4] [BZ#1048477] SUNRPC: remove KERN_INFO from dprintk() call sites
Bugzilla: 1048477
RH-Acked-by: Steve Dickson <SteveD@redhat.com>
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>

From: Chuck Lever <chuck.lever@oracle.com>

The use of KERN_INFO causes garbage characters to appear
when debugging is enabled.

Signed-off-by: Chuck Lever <chuck.lever@oracle.com>
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1048477
Upstream: 3a0799a94c0384a3b275a73267aaa10517b1bf7d
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/sunrpc/xprtrdma/transport.c b/net/sunrpc/xprtrdma/transport.c
index 285dc08..1eb9c46 100644
--- a/net/sunrpc/xprtrdma/transport.c
+++ b/net/sunrpc/xprtrdma/transport.c
@@ -733,7 +733,7 @@ static void __exit xprt_rdma_cleanup(void)
 {
  int rc;
 
- dprintk(KERN_INFO "RPCRDMA Module Removed, deregister RPC RDMA transport\n");
+ dprintk("RPCRDMA Module Removed, deregister RPC RDMA transport\n");
 #ifdef RPC_DEBUG
  if (sunrpc_table_header) {
   unregister_sysctl_table(sunrpc_table_header);
@@ -755,14 +755,14 @@ static int __init xprt_rdma_init(void)
  if (rc)
   return rc;
 
- dprintk(KERN_INFO "RPCRDMA Module Init, register RPC RDMA transport\n");
+ dprintk("RPCRDMA Module Init, register RPC RDMA transport\n");
 
- dprintk(KERN_INFO "Defaults:\n");
- dprintk(KERN_INFO "\tSlots %d\n"
+ dprintk("Defaults:\n");
+ dprintk("\tSlots %d\n"
   "\tMaxInlineRead %d\n\tMaxInlineWrite %d\n",
   xprt_rdma_slot_table_entries,
   xprt_rdma_max_inline_read, xprt_rdma_max_inline_write);
- dprintk(KERN_INFO "\tPadding %d\n\tMemreg %d\n",
+ dprintk("\tPadding %d\n\tMemreg %d\n",
   xprt_rdma_inline_write_padding, xprt_rdma_memreg_strategy);
 
 #ifdef RPC_DEBUG
-- 
1.7.1